__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    if (db.end) db.end();
```

Comment:
This does not need to be a conditional, refactor this to just invoke db.end on this line instead